src/Countries/US/Maryland/MarylandIncome/HasMarylandIncome.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Maryland\MarylandIncome;
+
+use Illuminate\Database\Eloquent\Collection;
+
+trait HasMarylandIncome
+{
+    public function getAdjustedEarnings()
+    {
+        return $this->getGrossEarnings() - $this->getStandardDeduction() - $this->getDependentExemption();
+    }
+
+    private function getStandardDeduction()
+    {
+        $amount = $this->getGrossEarnings() * self::STANDARD_DEDUCTION['percentange'];
+        if ($amount < self::STANDARD_DEDUCTION['min']) {
+            $amount = self::STANDARD_DEDUCTION['min'];
+        }
+        if ($amount > self::STANDARD_DEDUCTION['max']) {
+            $amount = self::STANDARD_DEDUCTION['max'];
+        }
+
+        return $amount;
+    }
+
+    private function getGrossEarnings()
+    {
+        // change if worked in delaware
+        $markup = $this->worked_in_delaware ? 1.032 : 1;
+        return $this->payroll->getEarnings() * $this->payroll->pay_periods * $markup;
+    }
+
+    private function getDependentExemption()
+    {
+        return 3200 * $this->tax_information->dependents;
```

Comment:
if 3200 and 1.032 are defined in the parent this trait could probably be reused next year unless they drastically change how taxes are calculated